travis-ci/cleanup-integration-tests.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+export AWS_ACCESS_KEY_ID="$INTEGRATION_AWS_ACCESS_KEY_ID"
+export AWS_SECRET_ACCESS_KEY="$INTEGRATION_AWS_SECRET_ACCESS_KEY"
+export AWS_DEFAULT_REGION="$INTEGRATION_AWS_DEFAULT_REGION"
+
+if [ -z "$DEPLOYMENT" ]; then
+  DEPLOYMENT=$(node ./travis-ci/select-stack.js)
+  if [ "$DEPLOYMENT" = "none" ]; then
+    echo "Unable to determine integration stack" >&2
+    exit 1
+  fi
+fi
+export DEPLOYMENT
+
+# Wait for the stack to be available
+KEY="travis-ci-integration-tests/${DEPLOYMENT}.lock"
+
+# Delete the stack if it's a nightly build
+if [ "$DEPLOYMENT" = "cumulus-nightly" ]; then
+  cd example
+  npm install
+
+  echo Delete app deployment
+
+  ./node_modules/.bin/kes cf delete \
+    --kes-folder app \
+    --region us-east-1 \
+    --deployment "$DEPLOYMENT" \
+    --yes
+
+  echo Delete iam deployment
+
+  ./node_modules/.bin/kes cf delete \
+    --kes-folder iam \
+    --region us-east-1 \
+    --deployment "$DEPLOYMENT" \
+    --yes
+
+  echo Delete app deployment
+
+  ./node_modules/.bin/kes lambda S3AccessTest delete \
+    --kes-folder app \
+    --region us-west-1 \
+    --deployment "$DEPLOYMENT" \
+    --yes
+fi
+
+# Release the stack
+DATE=$(date -R)
+STRING_TO_SIGN_PUT="DELETE
```

Comment:
should there be a closing `"` here?

Reply:
never-mind. I should have looked at the next line 🤦‍♂️

Reply:
that emojii looks funny in emacs:  
![screen shot 2018-09-13 at 11 58 20 am](https://user-images.githubusercontent.com/479480/45506413-5c601b80-b74c-11e8-9f9a-4a14356871c3.png)